src/base/bittorrent/session.cpp

```diff
@@ -439,6 +438,12 @@ Session::Session(QObject *parent)
     // Soon to be deprecated there
     // More info: https://github.com/arvidn/libtorrent/issues/2251
     pack.set_bool(libt::settings_pack::use_disk_cache_pool, false);
+    // libtorrent 1.1 enables UPnP & NAT-PMP by default
+    // turn them off before `libt::session` ctor to avoid split second effects
+    pack.set_bool(libt::settings_pack::enable_upnp, false);
+    pack.set_bool(libt::settings_pack::enable_natpmp, false);
+    pack.set_bool(libt::settings_pack::upnp_ignore_nonrouters, true);
+
```

Comment:
This blank line is redundant too.
The idea is settings_pack declared and filled in single code chunk.